tests/pass/shims/time-with-isolation.rs

```diff
@@ -0,0 +1,34 @@
+//@compile-flags: -Zmiri-disable-isolation
+
+use std::time::{Duration, Instant};
+
+fn duration_sanity(diff: Duration) {
```

Comment:
nit: It's unclear from this name what this function means (additionally, it's not exactly great to use terms like sanity in code, imo). It might be better just inlined into the caller.

Reply:
This file is a copy of `test.rs` minus the parts that use `SystemTime`.  I don't have a strong opinion about the name of the function but I'd change both if that was the case.